week7/src/components/Folder.js

```diff
@@ -0,0 +1,20 @@
+import "./Folder.css";
+
+export default function Folder({ user, folderName, owner }) {
+  const { profileImageSource, name } = owner || {};
+  //   const folderInfo = () => {
+  //     return <div></div>;
+  //   };
+
+  return (
+    <div className="folder">
+      <img
+        className="owner-img"
+        src={profileImageSource}
+        alt="프로필 이미지"
+      ></img>
+      <div>@{name}</div>
```

Comment:
텍스트를 위한 `p`태그를 사용해주세요 `div`는 구역을 나눌때 사용합니다.

```suggestion
      <p>@{name}</p>
```